package/network/utils/iproute2/patches/001-config.patch

```diff
@@ -1,7 +1,8 @@
 --- /dev/null
 +++ b/Config
-@@ -0,0 +1,4 @@
+@@ -0,0 +1,5 @@
 +# Fixed config to disable ATM support even if present on host system
 +TC_CONFIG_ATM:=n
 +TC_CONFIG_ACTION_GACT=y
 +TC_CONFIG_ACTION_PROB=y
```

Comment:
TC_CONFIG_ACTION_GACT=y and TC_CONFIG_ACTION_PROB=y do not exist any more.
Please add TC_CONFIG_XT:=y which would be added in case xtables is available